internal/hcp/registry/json.go

```diff
@@ -93,7 +93,19 @@ func (h *JSONRegistry) CompleteBuild(
 	artifacts []sdkpacker.Artifact,
 	buildErr error,
 ) ([]sdkpacker.Artifact, error) {
-	return h.bucket.completeBuild(ctx, build.Name(), artifacts, buildErr)
+	name := build.Name()
+	pluginMetadata := map[string]packer.PluginDetails{}
+	coreBuild, ok := build.(*packer.CoreBuild)
+	if ok {
+		pluginMetadata = coreBuild.GetPluginsMetadata()
+	}
+	for k, pluginDetails := range pluginMetadata {
+		fmt.Printf(
+			"[METADATA] JSON Metadata for build name %q: %q -- %q\n",
```

Comment:
This is bound to change, but I would suggest prefixing with something like `[TRACE]` here instead of `METADATA`. Also I'd suggest using `log.Printf` instead of `fmt.Printf` just so the logs are processed as we expect them to

Reply:
This is just during the test period so I can clearly search.